VideoLocker/res/values/styles.xml

```diff
@@ -42,6 +42,7 @@
         <item name="android:progressBarStyleInverse">@style/edX.Widget.ProgressBar.Inverse</item>
         <item name="android:progressBarStyleSmallInverse">@style/edX.Widget.ProgressBar.Small.Inverse</item>
         <item name="android:progressBarStyleLargeInverse">@style/edX.Widget.ProgressBar.Large.Inverse</item>
+        <item name="alertDialogTheme">@style/AppTheme.Dialog.Alert</item>
```

Comment:
Question: Should there be a corresponding `android: alertDialogTheme` as well ? (for the instance if we ever use an Alert Dialog other than the AppCompat one)